websites/A/AnimesWatch/presence.ts

```diff
@@ -0,0 +1,54 @@
+const presence = new Presence({
+		clientId: "972647321970016316",
+	}),
+	strings = presence.getStrings({
+		playing: "general.playing",
+		paused: "general.paused",
+		browsing: "general.browsing",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let video: HTMLVideoElement;
+
+presence.on("iFrameData", async (msg: HTMLVideoElement) => {
+	if (!msg) return;
+	video = msg;
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "animeswatch",
+		details: "Ana Sayfada Göz Gezdiriyor",
+		startTimestamp: browsingTimestamp,
+	};
+	if (document.location.pathname === "/") {
```

Comment:
Also changes the mentions of this
```suggestion
	},
	{ pathname } = document.location;
	if (pathname === "/") {
```